packages/firestore/test/integration/api/cursor.test.ts

```diff
@@ -233,4 +237,85 @@ apiDescribe('Cursors', persistence => {
         });
     });
   });
+
+  const makeTimestamp = (seconds, micros) =>
+    new Timestamp(seconds, micros * 1000);
+
+  it('can accept Timestamps as limits', () => {
```

Comment:
Not sure if this was copied from elsewhere, but since we have a limit() API, this would be clearer as "can accept Timestamps as bounds" or similar.